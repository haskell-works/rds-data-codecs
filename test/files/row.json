{
  "httpStatus": 200,
  "numberOfRecordsUpdated": 0,
  "records": [
    [
      {
        "longValue": 1234567890
      },
      {
        "longValue": 1
      },
      {
        "booleanValue": true
      },
      {
        "blobValue": "EjQ="
      },
      {
        "stringValue": "A"
      },
      {
        "stringValue": "AB"
      },
      {
        "stringValue": "C"
      },
      {
        "stringValue": "CD"
      },
      {
        "stringValue": "2024-02-04"
      },
      {
        "doubleValue": 3.14159265359
      },
      {
        "longValue": 42
      },
      {
        "stringValue": "{\"key\":\"value\"}"
      },
      {
        "stringValue": "{\"key\":\"value\"}"
      },
      {
        "stringValue": "1234.56"
      },
      {
        "stringValue": "12.34"
      },
      {
        "doubleValue": 3.14
      },
      {
        "longValue": 12345
      },
      {
        "longValue": 1
      },
      {
        "longValue": 1
      },
      {
        "stringValue": "Some text"
      },
      {
        "stringValue": "12:34:56"
      },
      {
        "stringValue": "12:34:56.780"
      },
      {
        "stringValue": "2024-02-04 12:34:56"
      },
      {
        "stringValue": "2024-02-04 12:34:56"
      },
      {
        "stringValue": "0123456789ABCDEFGHJKMNPQRS"
      },
      {
        "stringValue": "550e8400-e29b-41d4-a716-446655440000"
      }
    ]
  ]
}
