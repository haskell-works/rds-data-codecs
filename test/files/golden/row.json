[
    {
        "contents": 1234567890,
        "tag": "ValueOfInteger"
    },
    {
        "contents": 1,
        "tag": "ValueOfInteger"
    },
    {
        "contents": true,
        "tag": "ValueOfBool"
    },
    {
        "contents": "UldwUlBRPT0=",
        "tag": "ValueOfBase64"
    },
    {
        "contents": "A",
        "tag": "ValueOfText"
    },
    {
        "contents": "AB",
        "tag": "ValueOfText"
    },
    {
        "contents": "C",
        "tag": "ValueOfText"
    },
    {
        "contents": "CD",
        "tag": "ValueOfText"
    },
    {
        "contents": "2024-02-04",
        "tag": "ValueOfText"
    },
    {
        "contents": 3.14159265359,
        "tag": "ValueOfDouble"
    },
    {
        "contents": 42,
        "tag": "ValueOfInteger"
    },
    {
        "contents": "{\"key\":\"value\"}",
        "tag": "ValueOfText"
    },
    {
        "contents": "{\"key\":\"value\"}",
        "tag": "ValueOfText"
    },
    {
        "contents": "1234.56",
        "tag": "ValueOfText"
    },
    {
        "contents": "12.34",
        "tag": "ValueOfText"
    },
    {
        "contents": 3.14,
        "tag": "ValueOfDouble"
    },
    {
        "contents": 12345,
        "tag": "ValueOfInteger"
    },
    {
        "contents": 1,
        "tag": "ValueOfInteger"
    },
    {
        "contents": 1,
        "tag": "ValueOfInteger"
    },
    {
        "contents": "Some text",
        "tag": "ValueOfText"
    },
    {
        "contents": "12:34:56",
        "tag": "ValueOfText"
    },
    {
        "contents": "12:34:56.78",
        "tag": "ValueOfText"
    },
    {
        "contents": "2024-02-04 12:34:56",
        "tag": "ValueOfText"
    },
    {
        "contents": "2024-02-04 12:34:56",
        "tag": "ValueOfText"
    },
    {
        "contents": "550e8400-e29b-41d4-a716-446655440000",
        "tag": "ValueOfText"
    }
]